descqa/shear_test.py

```diff
@@ -120,7 +127,9 @@ def phi(self, lmax, n_z, cosmo, p, chi_recomb):
             phi_array.append(a)
         phi_array = np.array(phi_array)
         prefactor = 1.0  #(l+2)*(l+1)*l*(l-1)  / (l+0.5)**4
-        return l, phi_array * prefactor
+        #import healpy as hp
+        #pixwin = hp.pixwin(1024)[:lmax]
```

Comment:
I've just left this there as a convenience for the future because the DC2 shear maps are on a healpix grid, so the theory for that should include a pixelization window function. I was using this previously to look at the effect of varying levels of pixelization on the real-space analysis.